integration

```diff
@@ -656,6 +656,7 @@
   "kevinhaendel/ha-ubee",
   "kgn3400/state_updated",
   "killer0071234/ha-hiq",
+  "KipK/marees_france",
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>

Reply:
Anyway, I've added the country code.

Reply:
Oh, my bad.
With the name i just assumed it was france only.